src/hevm/test/test.hs

```diff
@@ -2239,7 +2239,7 @@ genNat :: Gen Int
 genNat = fmap fromIntegral (arbitrary :: Gen Natural)
 
 genName :: Gen Text
-genName = fmap T.pack $ listOf1 (oneof . (fmap pure) $ ['a'..'z'] <> ['A'..'Z'])
+genName = fmap (T.pack . ("esc_" <> )) $ listOf1 (oneof . (fmap pure) $ ['a'..'z'] <> ['A'..'Z'])
```

Comment:
maybe worth a comment here explaining what the `esc_` is for?

Reply:
Good point! I add an explanation :)